app/views/devise/mailer/confirmation_instructions.html.erb

```diff
@@ -0,0 +1,5 @@
+<p>Welcome <%= @email %>!</p>
```

Comment:
Maile konwertować do hamla, przetłumaczyć na polski jeżeli apka ma być po polsku, jeżeli po angielsku to zostawić, do dogadania z product ownerem